test_scripts/CloudAppRPCs/commonCloudAppRPCs.lua

```diff
@@ -121,6 +121,9 @@ local function ptu(self, app_id, ptu_update_func)
                 self.hmiConnection:SendResponse(d3.id, "BasicCommunication.SystemRequest", "SUCCESS", { })
                 self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate",
                   { policyfile = policy_file_path .. "/" .. policy_file_name })
+                if post_ptu_expectation then
+                  post_ptu_expectation(self)
+                end
```

Comment:
Due to changes to this file in the base branch, I think the best way to accomplish this is to copy the `policyTableUpdate` function from `actions.lua` and modify it to include this `post_ptu_expectation`